docs/user-guide/how-to-deploy.rst

```diff
@@ -0,0 +1,51 @@
+:desc: How to deploy your Rasa Assistant with Docker or Kubernetes
+
+.. _deploying-your-rasa-assistant:
+
+Deploying your Rasa Assistant
+===============================
+
+.. edit-link::
+
+This page explains when and how to deploy an assistant built with Rasa. 
+It will allow you to make your assistant available to users and set you up with a production-ready environment.
+If you haven't used Rasa before, we'd recommend that you start with the :ref:`rasa-tutorial`.
+
+.. contents::
+   :local:
+
+
+When to deploy your assistant
+--------------------------------
+
+The best time to deploy your assistant and make it available to test users is once it can handle the most important happy paths, or is what we call a "minimum viable assistant".
+
+The recommended deployment methods described below make it easy to share your assistant with test users via the `share your bot feature in Rasa X <../../rasa-x/docs/user-guide/enable-workflows#conversations-with-test-users>`_, even before selecting which channel you want to use. Then, when you’re ready to make your assistant available via :ref:`messaging-and-voice-channels`, you can easily add them to your existing deployment set up.
```

Comment:
```suggestion
The recommended deployment methods described below make it easy to share your assistant with test users via the `share your bot feature in Rasa X <../../rasa-x/docs/user-guide/enable-workflows#conversations-with-test-users>`_, even before deciding to put it live to your end users. Then, when you’re ready to make your assistant available via :ref:`messaging-and-voice-channels`, you can easily add them to your existing deployment set up.
```
"selecting which channel you want to use" sounds a bit confusing. 
Also is the feature called "share your bot" or "share your assistant"?

Reply:
changed to 'share your assistant' ; references elsewhere say that